controllers/rcj_soccer_referee_supervisor/referee/json_logger.py

```diff
@@ -0,0 +1,41 @@
+import json
+from pathlib import Path
+from datetime import datetime
+from typing import Optional
+
+
+class JSONLogger(object):
+    def __init__(self,
+                 logfile: Path):
```

Comment:
I kinda don't like this kind of formating. In this case the arguments fit to one line, so I would prefer to have them on the same line. If they do not fit, let's use
```python
def __init__(
    self,  # note only 4 spaces before the argument
    logfile: Path,
):
```

Reply:
One line it is!